charts/shoot-core/charts/podsecuritypolicies/templates/privileged-clusterrolebinding.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRoleBinding
+metadata:
+  name: garden.sapcloud.io:psp:privileged
+  annotations:
+    garden.sapcloud.io/description: |
+      Allow all authenticated users to use the privileged PSP.
+      The subject field is configured via .spec.kubernetes.allowPrivilegedContainers flag on the Shoot resource.
+      Do not manually change it as it'll be reconciled back to the original state.
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: garden.sapcloud.io:psp:privileged
+subjects:
+{{ if .Values.allowPrivilegedContainers -}}
```

Comment:
if this value is false we should set `subjects: []`, otherwise the `system:authenticated` subject won't be removed.